### DISCUSSION TITLE
Add description to controller input to remove confusion of players

### DISCUSSION THREAD

**Discussion by Smorty100** [ppy/osu#14539] [Ideas]
When I saw controller input at first I thought, that one could use the joystick to control the osu cursor but sadly that is not the case. I am sure many more people had the same idea and got disapointed afterwards and thus I request a description that says something the lines of "use the buttons of your controller to play taiko, mania or catch"

**Comment by bdach** [comment]
i am intensely confused. first off, why would you want to use the joystick to control the cursor? by "control" do you mean moving it? wouldn't that be horribly unwieldy?

secondly, where would this "description" you request even go?

**Reply by Smorty100** [reply]
Yes, I meant controlling the cursor movement. (Yes this would be horrible, but funny...) The description could be under the option to enable controller input. Just like in the other options, which have small grey descriptions under them.

**Reply by frenzibyte** [reply]
I honestly disagree with this. Ever since there was support for joysticks, there hasn't been any reports of "not being able to move the cursor with sticks", it's pointless to "clarify" that joysticks are meant for pressing buttons.

Also your proposed description is incorrect and unnecessarily limiting the joystick's true functionalities. Consider moving the catcher in osu!catch for example.

**Reply by Smorty100** [reply]
Oh... Then I am very sorry to bother you.
